application/src/test/java/com/ericsson/bss/cassandra/ecchronos/application/config/TestConfig.java

```diff
@@ -173,6 +174,29 @@ public void testAllValues() throws Exception
         assertThat(restServerConfig.getHost()).isEqualTo("127.0.0.2");
         assertThat(restServerConfig.getPort()).isEqualTo(8081);
     }
+    @Test
+    public void testInitialDelayDefaultValue() throws Exception
+    {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        File file = new File(classLoader.getResource("nothing_set.yml").getFile());
+
+        ObjectMapper objectMapper = new ObjectMapper(new YAMLFactory());
+
+        Config config = objectMapper.readValue(file, Config.class);
+        GlobalRepairConfig repair = config.getRepairConfig();
+        Interval interval = repair.getInitialDelay();
+        assertThat(interval.getInterval(TimeUnit.DAYS)).isEqualTo(1);
+    }
+
+    @Test
+    public void testIssue264_TestInitialDelayLongerThanRepairInterval() throws Exception
+    {
+        ClassLoader classLoader = Thread.currentThread().getContextClassLoader();
+        File file = new File(classLoader.getResource("intial_delay_greater_than_repair.yml").getFile());
```

Comment:
Why are you creating another file just to test a property?

Reply:
I am testing a faulty configuration file